preprocessing/nextclade/README.md

```diff
@@ -24,23 +24,21 @@ This preprocessing pipeline is still a work in progress. It requests unaligned n
    mamba env create -n loculus-nextclade -f environment.yml
    ```
 
-3. Start backend (see [backend README](../backend/README.md))
-4. Submit sequences to backend
+3. Start backend (see [backend README](../backend/README.md)), run ingest script to submit sequences from INSDC.
```

Comment:
Maybe easier to just use `./deploy.py` without the `--dev` arg and with ingest enabled but prepro disabled? Then it's a single command versus having to run backend separately in a terminal.

Reply:
Thanks!

```
(Alternatively you can run `./deploy.py --enablePreprocessing` to start the backend and preprocessing pods in one command.)
```